src/connect.js

```diff
@@ -22,6 +22,12 @@
  */
 function connect(params) {
   // write code here
+  return (extraParams) => (
+    {
+      ...params,
+      ...extraParams,
+    }
+  );
```

Comment:
не нужно переносить скобки
```suggestion
  return (extraParams) => ({
    ...params,
    ...extraParams,
  });
```